### ISSUE TITLE
Helm release installation fails for kubeapps with fluxcd/helm-controller #6303

### ISSUE THREAD

**Issue by project-administrator** [bitnami/charts#17115]
### Name and Version

bitnami/kubeapps 2.7.0

### What architecture are you using?

amd64

### What steps will reproduce the bug?

1. Install *fluxcd* with *helm-controller* running in the K8S cluster
2. Attempt to install the kubeapps with `fluxcd/helm-controller` e.g.:
```console
$ kubectl apply -f - <<EOF
apiVersion: source.toolkit.fluxcd.io/v1beta2
kind: HelmRepository
metadata:
  name: bitnami-repo-oci
  namespace: default
spec:
  interval: 6h
  url: oci://registry-1.docker.io/bitnamicharts
  type: oci
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: kubeapps
  namespace: default
spec:
  interval: 1h
  releaseName: kubeapps
  targetNamespace: default
  chart:
    spec:
      chart: kubeapps
      interval: 1m
      sourceRef:
        kind: HelmRepository
        name: bitnami-repo-oci
        namespace: default
EOF
```

### Are you using any custom parameters or values?

no

### What is the expected behavior?

Installation succeeds even if using the `fluxcd/helm-controller` instead of the command-line `helm install`.

### What do you see instead?

Checking the installation status reveals the following errors: 
```
$ kubectl describe helmrelease kubeapps
...
  Warning  error   97s (x8 over 3m21s)  helm-controller  Helm install failed: error while running post render on files: map[string]interface {}(nil): yaml: unmarshal errors:
  line 103: mapping key "initialDelaySeconds" already defined at line 97
  line 112: mapping key "initialDelaySeconds" already defined at line 106
```

### Additional information

Basically, helm chart generates an invalid template with duplicate keys in YAML. `fluxcd/helm-controller` would not tolerate this, but command-line `helm install` would silently remove the duplicate keys. 
Links to the same issue in the kubeapps and fluxcd projects: 
https://github.com/vmware-tanzu/kubeapps/issues/6303
https://github.com/fluxcd/helm-controller/issues/283

**Comment by carrodher** [comment]
Thanks for reporting this issue. Would you like to contribute by creating a PR to solve the issue? The Bitnami team will be happy to review it and provide feedback. [Here](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md) you can find the contributing guidelines.
